### PR TITLE
Add tags

### PR DESCRIPTION
**Pull request by piensaenpixel**
@javierarce could you take a look?

image

### CODE REVIEW

**Comment by piensaenpixel** [comment]
ping @javierarce

File: src/scss/cdb-components/tags.scss [hunk 1]
```diff
@@ -45,3 +46,93 @@
...
+  @include display-flex();
+}
+
+.CDB-ListMetadata-item {
+  margin-right: $baseSize;
+}
+
+.CDB-Metadata {
+  @include display-flex();
+  padding: $baseSize / 2 $baseSize / 2  $baseSize / 2 $baseSize;
```
Comment (javierarce) [inline src/scss/cdb-components/tags.scss]: I propose to do the following thing to make it easier to read:

```
$halfSize: $baseSize / 2;
padding: $halfSize $halfSize $halfSize $baseSize;
```

Comment (piensaenpixel) [inline src/scss/cdb-components/tags.scss]: Yeah and reuse that var for the rest, ok :dancer:

**Comment by piensaenpixel** [comment]
@javierarce add var take a look :)

**Comment by javierarce** [comment]
Let's find a better name for `.CDB-Metadata` and the `.CDB-ListDataType`, shall we?

I think both of them are actually tags (or CDB-Tags), but with a slightly different style and the possibility to have a close link.

We could then have:
- `CDB-ListMetadata`  and `.CDB-ListDataType` would be `CDB-ListTag'.
- `.CDB-ListMetadata-item` and `.CDB-ListDataType-item` would be `.CDB-ListTag-item`.
-  The `.CDB-Metadata` elements would be `.CDB-Tag.CDB-Tag--withLink` 
-  The `.CDB-DataType` elements would be `.CDB-Tag.CDB-Tag--opaque` (for example)

What do you think?

**Comment by javierarce** [comment]
:flags: Great!

**Pull request merged**